test/generator/metadata_tester_aottest.cpp

```diff
@@ -0,0 +1,397 @@
+#include "HalideRuntime.h"
```

Comment:
Ran this file through clang-format with our default settings; hopefully it's OK now.